packages/cli/src/index.ts

```diff
@@ -61,12 +61,20 @@ export async function main(medplumClient: MedplumClient, argv: string[]): Promis
 if (require.main === module) {
   dotenv.config();
   const baseUrl = process.env['MEDPLUM_BASE_URL'] || 'https://api.medplum.com/';
+  const fhirUrlPath = process.env['MEDPLUM_URL_FHIR_PATH'] || '';
```

Comment:
Rather than using two env vars, could this be a part of `MEDPLUM_BASE_URL`?

Reply:
I took a look, the FHIR endpoints might differ from auth endpoints. For example BCDA's endpoint will be (https://sandbox.bcda.cms.gov/auth/token) and it's FHIR endpoint will be https://sandbox.bcda.cms.gov/api/v2

Reply:
I'll take another look to see if we can use just one env vars.

Reply:
`fhirUrlPath` defaults to `fhir/r4`, we'll need to override this for the time being. I do agree, having the path be part of the baseUrl would be more convenient and similar to (https://github.com/google/medical_claims_tools)

Reply:
Agree that it needs to be 2 separate env vars.  

Can we rename `MEDPLUM_URL_FHIR_PATH` to `MEDPLUM_FHIR_URL_PATH`?  That way it's aligned with the current setting names:

```
baseUrl = MEDPLUM_BASE_URL
fhirUrlPath = MEDPLUM_FHIR_URL_PATH
```

Reply:
> Agree that it needs to be 2 separate env vars.
> 
> Can we rename `MEDPLUM_URL_FHIR_PATH` to `MEDPLUM_FHIR_URL_PATH`? That way it's aligned with the current setting names:
> 
> ```
> baseUrl = MEDPLUM_BASE_URL
> fhirUrlPath = MEDPLUM_FHIR_URL_PATH
> ```

 Good catch!